extensions/wikia/AnalyticsEngine/js/universal_analytics.js

```diff
@@ -347,6 +353,15 @@
     // Unleash
     _gaWikiaPush(['send', 'pageview']);
 
+    if (window.ads.context.opts.showAds) {
```

Comment:
I've added extra condition, but I'm not sure if it's possible. It's set at the php level - https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/AdEngine2ContextService.class.php#L45